src/main/java/com/revature/controller/ProfileController.java

```diff
@@ -63,8 +66,30 @@ public boolean insertProfileInfo(@RequestBody Profile profile) {
 	 the page to search for profiles.
      * @param page An integer identifier the page to search for profiles.
      * @return A list of all candidate profiles on a given page. */
-	public List<Profile> getAllPaged(@PathVariable int page) {
-		return profileService.getAllProfilesPaged(page);
+	public List<Profile> getAllPaged(@PathVariable int page,
+			@RequestParam("filtervalue") String value, @RequestParam("skillids") String data) {
+ 		if (!value.isEmpty() || !data.isEmpty()) {

```

Comment:
Can be changed to be more clear -> !(value.isEmpty() && data.isEmpty()) which means if they are not both empty. Much clearer. Additionally, value is not a very clear variable name. Preferably change it to something more telling of what it contains.

Reply:
fixed logic

Reply:
logic resolved